src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -895,77 +1127,204 @@ public static SlhDsa ImportEncryptedPkcs8PrivateKey(ReadOnlySpan<char> password,
+        ///     The base-64 decoded contents of the PEM text from <paramref name="source" />
+        ///     do not represent an ASN.1-BER-encoded PKCS#8 EncryptedPrivateKeyInfo structure.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     The base-64 decoded contents of the PEM text from <paramref name="source" />
+        ///     indicate the key is for an algorithm other than the algorithm
+        ///     represented by this instance.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     The base-64 decoded contents of the PEM text from <paramref name="source" />
+        ///     represent the key in a format that is not supported.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     An error occurred while importing the key.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="PlatformNotSupportedException">
+        ///   The platform does not support SLH-DSA. Callers can use the <see cref="IsSupported" /> property
+        ///   to determine if the platform supports SLH-DSA.
         /// </exception>
+        /// <remarks>
+        ///   <para>
+        ///     When the base-64 decoded contents of <paramref name="source" /> indicate an algorithm that uses PBKDF1
+        ///     (Password-Based Key Derivation Function 1) or PBKDF2 (Password-Based Key Derivation Function 2),
+        ///     the password is converted to bytes via the UTF-8 encoding.
+        ///   </para>
+        ///   <para>
+        ///     Unsupported or malformed PEM-encoded objects will be ignored. If multiple supported PEM labels
+        ///     are found, an exception is thrown to prevent importing a key when
+        ///     the key is ambiguous.
+        ///   </para>
+        ///   <para>This method supports the <c>ENCRYPTED PRIVATE KEY</c> PEM label.</para>
+        /// </remarks>
         public static SlhDsa ImportFromEncryptedPem(ReadOnlySpan<char> source, ReadOnlySpan<char> password)
         {
             ThrowIfNotSupported();
 
-            // TODO: Match the behavior of ECDsa.ImportFromEncryptedPem.
-            throw new NotImplementedException();
+            return PemKeyHelpers.ImportEncryptedFactoryPem<SlhDsa, char>(
+                source,
+                password,
+                ImportEncryptedPkcs8PrivateKey);
         }
 
         /// <summary>
-        ///   Imports an SLH-DSA key from an RFC 7468 PEM-encoded string.
+        ///   Imports an SLH-DSA key from an encrypted RFC 7468 PEM-encoded string.
         /// </summary>
         /// <param name="source">
-        ///   The text of the PEM key to import.
-        /// </param>
+        ///   The PEM text of the encrypted key to import.</param>
         /// <param name="passwordBytes">
-        ///  The password to use when decrypting the key material.
+        ///   The bytes to use as a password when decrypting the key material.
```

Comment:
Looks like the comment got changed to "bytes", but this one is a ROS-char, so revert.